eng/illink.targets

```diff
@@ -57,19 +60,31 @@
   -->
   <ItemGroup Condition="'$(BinPlaceILLinkTrimAssembly)' == 'true'">
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/trimmed</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/trimmed</RuntimePath>
       <ItemName>TrimmedItem</ItemName>
     </BinPlaceTargetFramework>
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/reports</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/reports</RuntimePath>
       <ItemName>TrimmingReport</ItemName>
     </BinPlaceTargetFramework>
     <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/pretrimmed</RuntimePath>
+      <RuntimePath>$(ILLinkTrimAssemblyArtifactsRootDir)/pretrimmed</RuntimePath>
```

Comment:
Once you update ILLinkTrimAssemblyArtifactsRootDir to use NormalizeDirectory, remove `/` from these usages.